include/pmacc/simulationControl/TimeInterval.hpp

```diff
@@ -46,9 +46,9 @@ namespace pmacc
          */
         static double getTime()
         {
-            struct timeval act_time;
-            gettimeofday(&act_time, nullptr);
-            return (double)act_time.tv_sec*1000. + (double)act_time.tv_usec / 1000.;
```

Comment:
QUESTION:
@ComputationalRadiationPhysics/picongpu-developers 
Does `usec` mean µs? Is this is the case, we have underestimated the time.

Reply:
I think that was fine: uses was nano seconds, so multiply by 1000 to get milliseconds.

Reply:
actually I think the old one was wrong.
tv_sec divided by 1000 would have been ms and usec multiplied by 1000 would also have been ms.

but it did it exactly the other way around, no?

Reply:
ah no wait, I am still tired. all was fine and is fine